### PR TITLE
[NUI][TCSACR-335] Delete the guides of Popup, Tab and Toast

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Change Description ###

1. Delete the guides of Popup, Tab and Toast
2. Change StateChangeEvent to ControlStateChangeEvent in Button, RadioButton, CheckBox and Slider

### API Changes ###
 - ACR: https://code.sec.samsung.net/jira/browse/TCSACR-335

### CODE REVIEW

**Comment by mijin85cho** [comment]
Question.
According to the https://code.sec.samsung.net/jira/browse/TCSACR-335, 
StateChangedEvent in Button and Slider are also deprecated.

However when I searched StateChangedEvent, I can see they still exist like below.
 - https://github.com/Samsung/tizen-docs/blob/tizen_6.0_prepare/docs/application/dotnet/guides/nui/nui-components/Button.md#responding-to-statechangedevent
 - https://github.com/Samsung/tizen-docs/blob/tizen_6.0_prepare/docs/application/dotnet/guides/nui/nui-components/Slider.md#responding-to-statechangedevent

Don't we need to update these pages also?

**Comment by zhouleonlei** [comment]
> Question.
> According to the https://code.sec.samsung.net/jira/browse/TCSACR-335,
> StateChangedEvent in Button and Slider are also deprecated.
> 
> However when I searched StateChangedEvent, I can see they still exist like below.
> 
> * https://github.com/Samsung/tizen-docs/blob/tizen_6.0_prepare/docs/application/dotnet/guides/nui/nui-components/Button.md#responding-to-statechangedevent
> * https://github.com/Samsung/tizen-docs/blob/tizen_6.0_prepare/docs/application/dotnet/guides/nui/nui-components/Slider.md#responding-to-statechangedevent
> 
> Don't we need to update these pages also?

Sorry for missing, I modified Button, CheckBox, RadioButton and Slider

File: docs/application/dotnet/guides/nui/nui-components/Button.md [hunk 1]
```diff
@@ -160,11 +160,11 @@ When you change the Button state to focus or disable, the Button instance receiv
 
 ```cs
 Button button = new Button();
-button.StateChangedEvent += OnStateChange;
+button.ControlStateChangedEvent += OnStateChange;
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/nui-components/Button.md:163]: Just for checking. Would you please confirm "**button**.ControlStateChangedEvent" here?
I'm confused as "View.ControlStateChangedEvent" is said to replace StateChangedEvent  in Button.

Comment (zhouleonlei) [reply docs/application/dotnet/guides/nui/nui-components/Button.md:163]: > Just for checking. Would you please confirm "**button**.ControlStateChangedEvent" here?
> I'm confused as "View.ControlStateChangedEvent" is said to replace StateChangedEvent in Button.

The class of "View" is the parent class of "Button" and "ControlStateChangedEvent"  is public, So we can use button.ControlStateChangedEvent here

**Review by mijin85cho — APPROVED** [review]
Thanks for the updates.

**Pull request merged**